shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java

```diff
@@ -57,6 +58,9 @@ public void prepareDependencies() throws IOException {
     mMetaMasterClient = mock(MetaMasterClient.class);
     MasterInfo masterInfo = MasterInfo.newBuilder()
         .setLeaderMasterAddress("testAddress")
```

Comment:
add port information? like "testAddress:19998" to similar to local test results?